__tests__/app.test.js

```diff
@@ -92,6 +92,115 @@ describe('Get /api/reviews', () => {
                 expect(reviews).toBeSortedBy("created_at", {descending: true,});
             });
     });
+
+    test("200: response with array of all reviews of the correct category", () => {
+        return request(app)
+            .get("/api/reviews?category=social-deduction&sort_by=created_at&order=ASC")
+            .expect(200)
+            .then( res => {
+                const { reviews } = res.body;
+                expect(reviews).toBeInstanceOf(Array);
+                reviews.forEach((review) => {
+
+                    expect(review).toEqual(
+                        expect.objectContaining({
+                            owner: expect.any(String),
+                            title: expect.any(String),
+                            review_id: expect.any(Number),
+                            category: "social deduction",
+                            review_img_url: expect.any(String),
+                            created_at: expect.any(String),
+                            votes: expect.any(Number),
+                            designer: expect.any(String),
+                            comment_count: expect.any(Number)
+                        })
+                    );
+                });
+                expect(reviews).toBeSortedBy("created_at", {ascending: true,});
+            });
+    });
+
+    test("200: response sorted and ordered by queries passed", () => {
+        return request(app)
+            .get("/api/reviews?category=social-deduction&sort_by=review_id&order=ASC")
+            .expect(200)
+            .then( res => {
+                const { reviews } = res.body;
+                expect(reviews).toBeSortedBy("review_id", {ascending: true,});
+            });
+    });
+
+    test("200: response with array of all reviews if category omitted", () => {
+        return request(app)
+            .get("/api/reviews?sort_by=created_at&order=DESC")
+            .expect(200)
+            .then( res => {
+                const { reviews } = res.body;
+                expect(reviews).toBeInstanceOf(Array);
```

Comment:
This assertion does not need to be made as the below .foreach() does this for you.